src/features/settings/appearance/CompactSettings.tsx

```diff
@@ -18,6 +18,12 @@ export default function CompactSettings() {
   const { thumbnailsPosition, showVotingButtons, thumbnailSize } =
     useAppSelector((state) => state.settings.appearance.compact);
 
+  const postsAppearanceType = useAppSelector(
+    (state) => state.settings.appearance.posts.type,
+  );
+
+  if (!(postsAppearanceType === "compact")) return <></>;
```

Comment:
Lets still render if compact mode - its what apollo does :)

Reply:
You mean, when compact mode is off?

Reply:
Yeah, I would like to keep the previous behavior where the compact settings show all times.

There will probably be a feature in the future where you can change compact to large post mode in the feed.